teams/wg-llvm.toml

```diff
@@ -6,6 +6,9 @@ wg = true
 leads = ["nagisa"]
 members = ["nagisa"]
```

Comment:
Did you intend to extend the members too?

Reply:
Oh I completely didn't look at the members list, I definitely think we should do that! I think adding @nikic and @cuviper is right? Are there others who should be included?

Reply:
Three of us make a good start, better than putting it all on @nagisa. :slightly_smiling_face: